2008-05-21  Ulrich Drepper  <drepper@redhat.com>

	* iconvdata/Depend: Add localedata.

2008-05-14  Joseph Myers  <joseph@codesourcery.com>

	* iconvdata/Makefile (bug-iconv6-ENV, tst-iconv7-ENV): Define.

2008-05-14  Ulrich Drepper  <drepper@redhat.com>

	* iconvdata/bug-iconv6.c (do_test): Use de_DE.UTF-8 locale.

diff a/iconvdata/Depend b/iconvdata/Depend
--- a/iconvdata/Depend
+++ b/iconvdata/Depend
@@ -1 +1,2 @@
 iconv
+localedata
diff a/iconvdata/Makefile b/iconvdata/Makefile
--- a/iconvdata/Makefile
+++ b/iconvdata/Makefile
@@ -73,6 +73,9 @@ tests += bug-iconv3
 endif
 
 test-srcs := tst-table-from tst-table-to
+
+bug-iconv6-ENV = LOCPATH=$(common-objpfx)localedata
+tst-iconv7-ENV = LOCPATH=$(common-objpfx)localedata
 endif
 
 # No code here is in libc.so.
diff a/iconvdata/bug-iconv6.c b/iconvdata/bug-iconv6.c
--- a/iconvdata/bug-iconv6.c
+++ b/iconvdata/bug-iconv6.c
@@ -13,7 +13,7 @@ static const char testbuf[] = {
 static int
 do_test (void)
 {
-  setlocale (LC_ALL, "en_US.UTF-8");
+  setlocale (LC_ALL, "de_DE.UTF-8");
   iconv_t ic = iconv_open ("ISO-2022-JP//TRANSLIT", "UTF-8");
   if (ic == (iconv_t) -1)
     {
